pkg/server/metrics.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/google/go-github/github"
+)
+
+var (
+	rateLimit = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "github_rate_limits",
+		Help: "Rate limit for GitHub API calls",
+	}, []string{"type"})
+
+	webhookCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "webhook_counter",
+		Help: "A counter of the webhooks made to service.",
+	}, []string{"event_type"})
+
+	handledEventsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "handled_events_counter",
+		Help: "A counter of handled events.",
+	}, []string{"event_type"})
+)
+
+func init() {
```

Comment:
instead of initializing it immediately name it for example `registerMetrics` and call from the `plugin_init.go`